File: app/src/main/java/com/school/optimstudy/study_tracker_activity/StudyTracker.java
package com.school.optimstudy.study_tracker_activity;

import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.school.optimstudy.R;

public class StudyTracker extends Fragment {
    public static StudyTracker newInstance() {
        return new StudyTracker();
    }

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.activity_study_tracker, container, false);
    }
}

File: app/src/main/java/com/school/optimstudy/study_tracker_activity/StudyTracker.kt
package com.school.optimstudy.study_tracker_activity

import android.support.v4.app.Fragment
import android.os.Bundle
import android.support.v7.widget.LinearLayoutManager
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.school.optimstudy.R
import kotlinx.android.synthetic.main.activity_study_tracker.*
import kotlinx.android.synthetic.main.activity_study_tracker.view.*

class StudyTracker : Fragment() {
    companion object {
        fun newInstance(): StudyTracker {
            return StudyTracker()
        }
    }

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater!!.inflate(R.layout.activity_study_tracker, container, false)
    }

    override fun onViewCreated(view: View?, savedInstanceState: Bundle?) {
        val data: ArrayList<Pair<String, String>> = arrayListOf(
                Pair("Physics", "1:30:45"),
                Pair("Intro to SWE", "1:30:45"),
                Pair("User Centered Design", "1:30:45")
        )

        timers_recycler.apply {
            setHasFixedSize(true)
            layoutManager = LinearLayoutManager(context)
            adapter = StudyTrackerRecycleAdapter(data)
        }
    }
}
